### PR TITLE
Updated version of the pipeline making perl script

### PR DESCRIPTION
**Pull request by CAUSA**

### CODE REVIEW

**Comment by CAUSA** [comment]
Close the pull request to resolve a conflict in README.md

**Comment by CAUSA** [comment]
The conflict is caused by a mistakenly rename of the 0README file to README.md. 
I have replaced it with the original README.md file, but there are still conflicts in both 0README and README.md.
 Please help to resolve the conflicts. I am not able to do it.

**Comment by rtraborn** [comment]
Hi @CAUSA:

Don't worry- this shouldn't be a problem to fix. 
More importantly for now: please see the review comments I made and let me know what you think.

**Comment by CAUSA** [comment]
where can I find the review comments?

**Comment by rtraborn** [comment]
It looks like they were automatically deleted when you unexpectedly closed the pull request. I'll (re-)write them here in the comments so the same thing doesn't happen again.

**Comment by CAUSA** [comment]
I am sorry for that. I did not see your comments and I did not know that you are writing sth. I closed it almost the same time when you are writing comments. This might be a bug of GitHub.

**Comment by rtraborn** [comment]
Hi @CAUSA:

No worries. Here's the code review I wrote previously.

1. Can you provide a minimal (ie not running on an entire population; just a few clones) reproducible example in the `README` section? 
2. I can't find a list of required software anywhere. This could be added in `INSTALL.txt` (which could itself be changed to `INSTALL.md` for the purposes of formatting) 
3. I'm ok in principle with the approach you're using here: use a perl wrapper to create a number of pbs scripts that are then executed by a shell script. However, can you please state/summarize this at the beginning of the `README`? It's not intuitive to me when reading the documentation until later.
4. Instead of asking the user to change the input file names, why can't we make use of the sample names that exist on the file system? One solution could be to add a script that creates symbolic links as appropriate.
5. How does one set the flags for eg bwa using this pipeline? If they have to be hard-coded, then I suggest adding a `.conf` file as in: [https://github.com/rtraborn/GoRAMPAGE/blob/master/scripts/GoRAMPAGE.conf]

Let me know if you have any questions- we can always discuss this in person if you'd prefer.
Cheers,
Taylor

**Comment by CAUSA** [comment]
Hi, @rtraborn,
Your suggestions are surely good and would. Just takes time to do it.
These are good enough for my personal use, but when publishing here, we should make it as better as possible. If you would, you may change them as you wish. I would like to discuss this with you in person at anytime.

**Comment by CAUSA** [comment]
Hi @rtraborn,
You may do it in any way you want. In fact, I do not use Perl very often. I do programming mostly in Java and C. This Perl script is based on a Perl script Zhiqiang send to me. In order to save time, I just made some modifications to make it more powerful and clear.

**Comment by rtraborn** [comment]
Hi @CAUSA: my previous comment was not a knock on perl (or this approach) at all! I was just mentioning how I would go about doing it given my experience and expertise. There's not a single right or wrong way to go about this.
Since you've made a few commits since Tuesday, can you summarize the changes/updates you've made in the interim period here before we go any further? 
Thanks for all of your hard work!
Taylor

**Comment by CAUSA** [comment]
Hi @rtraborn, I know but I mean I myself am not familiar with Perl either, and I do not like Perl very much. Very small changes are made in this week, to fix some bugs and to make the notification/information more accurate.
Thanks!
Xiaolong

**Comment by rtraborn** [comment]
Hi @CAUSA: I'm happy to merge this but the conflicts in `0README` and `README.md` need to be resolved first. Let me know if you need help with this.

**Comment by CAUSA** [comment]
Hi @rtraborn,
I updated the files just because Takahiro asked me to send the new protocol to him. 

Xiaolong
